### DISCUSSION TITLE
[WRN] [CVE-2019-6799] Malformed version: [710 2 73823375 488 2960455]

### DISCUSSION THREAD

**Discussion by xgt6op** [projectdiscovery/nuclei#3949] [Q&A]
Hello I am getting this while running the nuclei scan on a subdomain list .

[WRN] [CVE-2019-6799] Malformed version: [17 6.20.35 188]
[WRN] [CVE-2019-6799] Malformed version: [6.20.35 188 17]
[WRN] [CVE-2019-6799] Malformed version: [6.20.35 188 17]
[WRN] [CVE-2019-6799] Malformed version: [17 6.20.35 188]
[WRN] [CVE-2019-6799] Malformed version: [6.20.35 188 17]
[WRN] [CVE-2019-6799] Malformed version: [6.20.35 188 17]
[WRN] [CVE-2019-6799] Malformed version: [710 2 73823375 488 2960455]
[WRN] [CVE-2019-6799] Malformed version: [73823375 488 2960455 710 2]
[WRN] [CVE-2019-6799] Malformed version: [73823375 488 2960455 710 2]
[WRN] [CVE-2019-6799] Malformed version: [710 2 73823375 488 2960455]
[WRN] [CVE-2019-6799] Malformed version: [488 2960455 710 2 73823375]
[WRN] [CVE-2019-6799] Malformed version: [488 2960455 710 2 73823375]

Is there any solution or is this my problem only?

**Comment by MetzinAround** [comment]
Hi! What command and template did you use for this?

**Reply by xgt6op** [reply]
nuclei -l live_subdomains -es info -t /home/xgt6op/nuclei-templates/http/cves/2019/CVE-2019-6799.yaml -o output2.txt

**Reply by MetzinAround** [reply]
what's in your live_subdomains list? Is it a custom one you made or generic

**Reply by MetzinAround** [reply]
@ehsandeep The [WRN] here don't seem like complete errors to me, but I'm confused about the output on this. Could you take a look?

**Reply by xgt6op** [reply]
in my subdomains list there are subdomains which has been generated by httpx.

**Comment by geeknik** [comment]
I am seeing this as well with nuclei v2.9.9 and nuclei-templates v9.5.8. My command line was `nuclei -l test-subs.txt -o test-out.txt`:
```
[WRN] [CVE-2019-12616] Malformed version: [1613518894 1613518970 166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357 178885259474031826261567127339 2 20220906]
[WRN] [CVE-2019-6799] Malformed version: [12001839194546984181565804357 20220906 94224023136283657951565804356 2 1567538982 1567538110 1613518970 178885259474031826261567127339 1567538732 166284507838492898991674766505 1613518894 1613526560 1567610209 1567538499 1567610119]
[WRN] [CVE-2019-6799] Malformed version: [2 20220906 1613518894 1613518970 166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357 178885259474031826261567127339]
[WRN] [CVE-2019-6799] Malformed version: [178885259474031826261567127339 2 20220906 1613518894 1613518970 166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357]
[WRN] [CVE-2019-6799] Malformed version: [2 20220906 1613518894 1613518970 166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357 178885259474031826261567127339]
[WRN] [CVE-2019-6799] Malformed version: [166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357 178885259474031826261567127339 2 20220906 1613518894 1613518970]
[WRN] [CVE-2019-6799] Malformed version: [1613518894 1613518970 166284507838492898991674766505 94224023136283657951565804356 12001839194546984181565804357 178885259474031826261567127339 2 20220906]
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
